frontend/packages/kubevirt-plugin/integration-tests/tests/models/baseVirtualMachine.ts

```diff
@@ -78,6 +80,8 @@ export class BaseVirtualMachine extends KubevirtUIResource<VMBuilderData> {
           VM_STATUS.Running,
           resolveTimeout(timeout, VM_BOOTUP_TIMEOUT_SECS),
         );
+        // wait for the VM to boot up
+        execSync(`expect ${EXPECT_LOGIN_SCRIPT_PATH} ${this.name} ${this.namespace}`);
```

Comment:
I am getting this

```
 Error: Command failed: expect ./packages/kubevirt-plugin/integration-tests/tests/utils/scripts/expect-login.sh vm-list-actions-test-ayeaf test-ayeaf
   couldn't read file "./packages/kubevirt-plugin/integration-tests/tests/utils/scripts/expect-login.sh": no such file or directory
```

we probably need a better path resolution

Reply:
also the expect-login script was not meant for checking(expecting) anything originally. Only for waiting for successful boot. 

That means it will not fail if the VM does not boot succesfully. We need return non null return code by the script (eg https://stackoverflow.com/a/58222144)

Reply:
also are we planning to add expect-login to other actions as well? like restart/migrate/ etc. ?

Reply:
Fixed expect-login script and move it into `waitForStatus`, so other actions which has start action will have the check as well.

Reply:
I fixed the path on my machine, since I had it misconfigured.
But nevertherless, it is quite hardcoded for my taste (would be better to resolve the working directory on runtime). We can let it be for now...